examples/access_traits/example_cuda_access_traits.cpp

```diff
@@ -67,12 +67,16 @@ int main(int argc, char *argv[])
 
     cudaMemcpy(d_a, a.data(), sizeof(a), cudaMemcpyHostToDevice);
 
-    using device_type = Kokkos::Cuda::device_type;
-    ArborX::BVH<device_type> bvh{PointCloud{d_a, d_a, d_a, N}};
+    using MemorySpace = Kokkos::CudaSpace;
+    using ExecutionSpace = Kokkos::Cuda;
+    ArborX::BVH<MemorySpace> bvh{ExecutionSpace{},
+                                 PointCloud{d_a, d_a, d_a, N}};
 
-    Kokkos::View<int *, device_type> indices("indices", 0);
-    Kokkos::View<int *, device_type> offset("offset", 0);
-    bvh.query(Spheres{d_a, d_a, d_a, d_a, N}, indices, offset);
+    using DeviceType = ExecutionSpace::device_type;
+    Kokkos::View<int *, DeviceType> indices("indices", 0);
+    Kokkos::View<int *, DeviceType> offset("offset", 0);
+    bvh.query(ExecutionSpace{}, Spheres{d_a, d_a, d_a, d_a, N}, indices,
```

Comment:
What about reusing the `ExecutionSpace` instance?

Reply:
There is little point w/o using CUDA streams